lib/access_import/accessors/accessor.rb

```diff
@@ -16,11 +16,17 @@ def add_other_accessors(accessors)
     end
   end
 
+  def add_records_hash
+    @records = hash_all unless @records
```

Comment:
I implemented that, because that way I do not need to worry about how many and what Accor class childs I initialize. If their not accessed, it will not slow down the whole thing so much.

> `@all ||= hash_all`

But this will not keep the "hashed" access table stored in the class. So my accessor needs to hash the table every time I'm accessing it.
Some of them are accessed by .all, others use `find(primary_key)`
I'm not sure if that is a good idea in general. But It worked out so far. ..